src/components/ClientReservationListItem.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from "react";
+import { View, FlatList, RefreshControl } from "react-native";
+import { ListItem, ButtonGroup } from "react-native-elements";
+import { connect } from "react-redux";
+import { NavigationActions } from "react-navigation";
+import { MONTHS, DAYS, MAIN_COLOR } from "../constants";
+import { Spinner, EmptyList } from "./common";
+import { onClientReservationListRead } from "../actions";
+import moment from "moment";
+
+class ClientReservationListItem extends Component {
+  constructor(props) {
+    super(props);
+    props.onClientReservationListRead();
+    this.state = {
+      selectedIndex: 1,
+      filteredList: []
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.reservations !== this.props.reservations) {
+      this.updateIndex(this.state.selectedIndex);
+    }
+  }
+
+  OnPressItem = reservation => {
+    const navigateAction = NavigationActions.navigate({
+      routeName: "reservationsDetail",
+      params: reservation
+    });
+    this.props.navigation.navigate(navigateAction);
```

Comment:
lo anterior tambien se puede hacer asi y no tenes que importar NavigationActions:
this.props.navigation.navigate('reservationDetail', { reservation })

pd: no es que este mal, nomas propongo la alternativa

Reply:
Cual es mejor?

Reply:
para mi la forma que te digo yo, es mas simple y te ahorras un import